core/tee/tee_svc_storage.c

```diff
@@ -932,9 +932,9 @@ TEE_Result syscall_storage_obj_trunc(unsigned long obj, size_t len)
 			if (res != TEE_SUCCESS)
 				goto exit;
 			res = TEE_ERROR_CORRUPT_OBJECT;
-			goto exit;
```

Comment:
this `goto exit` is useless since the once added line 937.

Note: I think the following way is easier to read (maybe i'm a bit lazy):
```
	res = o->pobj->fops->truncate(o->fh, len + off);
	switch (res) {
	case TEE_SUCCESS:
		o->info.dataSize = len;
		break;
	case TEE_ERROR_CORRUPT_OBJECT:
		EMSG("Object corruption");
		(void)tee_svc_storage_remove_corrupt_obj(sess, o);
		break;
	default:
		res = TEE_ERROR_GENERIC;
		break;
	}

exit:
	return res;
}
```